internal/tiltfile/tiltfile_state.go

```diff
@@ -351,6 +363,7 @@ const (
 	k8sKindN                    = "k8s_kind"
 	k8sImageJSONPathN           = "k8s_image_json_path"
 	workloadToResourceFunctionN = "workload_to_resource_function"
+	customDeployN               = "custom_deploy"
```

Comment:
we should probably put kubernetes/k8s in the name somewhere? since it's expected to return k8s yaml

Reply:
That's a great point - `k8s_custom_deploy` seem reasonable?

Reply:
:+1: